<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arriwal</class>
 <widget class="QWidget" name="arriwal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 103, 73);
color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>30</y>
     <width>501</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>120</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во экземпляров:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tabArriwal">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Таблица Книги</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить поле</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tabProdBtn">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>320</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Таблица Продажи</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameTable">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер документа:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сведения о поставщике:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
